openhtf/plugs/usb/__init__.py

```diff
@@ -36,9 +38,27 @@ def MyPhase(test, adb):
 conf.Declare('usb_server', 'USB Server IP/Hostname')
 conf.Declare('usb_server_port', 'USB Server Port',
              default_value=10000)
-
 conf.Declare('libusb_rsa_key', 'A private key file for use by libusb auth.')
+conf.Declare('remote_usb', 'Ethersyncy or other',default_value=None)
```

Comment:
The docstring for remote_usb could be more descriptive. Something like "Type of provider of remote USB connectivity." Then you could add text to the module docstring to show an example and list the choices (right now there will only be 'ethersync'.